geometry_msgs/PoseStamped[] goals
bool loop
---
bool success
string message